certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1145 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DomainTools Iris Enrich",
+    "description": "Receive immediate context to indicators to bolster triage, response, and defense.",

```

Comment:
@RamboV Can you please provide some more details of your connector which should describe concisely the main purpose and value offered by your connector?

Reply:
Hi @sriyen-msft , Modified the descriptions